api/client/clientModel.js

```diff
@@ -60,6 +59,46 @@ function updateDelete(id) {
     });
 }
 
+// FOR CLIENT INTAKE ROUTES
+
+const updatePhoneNumbersByClientId = (id, changes) => {
+  return db('phone_numbers').where({ client_id: id }).update(changes, '*');
+};
+
+const updateEmailByClientId = (id, changes) => {
+  return db('email_addresses').where({ client_id: id }).update(changes, '*');
+};
+
+const updateEducationHistoryByClientId = (id, changes) => {
+  return db('education_histories')
+    .where({ client_id: id })
+    .update(changes, '*');
+};
+
+const getGenderIdByClientId = async (id) => {
+  return db('genders').where({ client_id: id }).first;
+};
+
+const getRaceIdByClientId = async (id) => {
+  return db('races').where({ client_id: id }).first;
+};
+
+const getEthnicityByClientId = async (id) => {
```

Comment:
see comment for genders, `ethnicities` table is a lookup table and not related to client_id.